app/uploaders/profile_photo_attachment_uploader.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class ProfilePhotoAttachmentUploader < CarrierWave::Uploader::Base
+  include ValidateFileSize
+  include SetAwsConfig
+
+  MAX_FILE_SIZE = 10.megabytes
+
+  def initialize(guid, form_id)
+    @guid = guid
+    @form_id = form_id
+
+    super(@guid)
+
+    if Rails.env.production?
+      set_aws_config(
+        Settings.vic.s3.aws_access_key_id,
+        Settings.vic.s3.aws_secret_access_key,
+        Settings.vic.s3.region,
+        Settings.vic.s3.bucket
+      )
+
+      self.aws_acl = 'public-read'
```

Comment:
Do we need `public-read` here now?

Reply:
@patrickvinograd - I think this means we're going to need to authenticate requests to S3 for this bucket at the revproxy layer (http://s3.amazonaws.com/doc/s3-developer-guide/RESTAuthentication.html).

Reply:
Agreed, but I'm hoping we can give an IAM role to the rev proxy instances instead of passing the REST auth header. I don't know if that's actually the case yet though

Reply:
Was thinking about rev proxy getting content, but of course this piece is for vets-api instances putting content. Same comment applies though.